Configurations/10-main.conf

```diff
@@ -6,18 +6,21 @@ my $vc_win64a_info = {};
 sub vc_win64a_info {
     unless (%$vc_win64a_info) {
         if (`nasm -v 2>NUL` =~ /NASM version ([0-9]+\.[0-9]+)/ && $1 >= 2.0) {
-            $vc_win64a_info = { as        => "nasm",
-                                asflags   => "-f win64 -DNEAR -Ox -g",
-                                asoutflag => "-o" };
+            $vc_win64a_info = { AS        => "nasm",
+                                ASFLAGS   => "-g",
+                                asflags   => "-Ox -f win64 -DNEAR",
+                                ASOUTFLAG => "-o" };
```

Comment:
This line caught my eye. Even though `-ofile.obj` apparently works, manual page says that it's `-o file.obj`. It might be appropriate to add space...

Reply:
Okie